vllm/config.py

```diff
@@ -165,7 +188,8 @@ def _verify_quantization(self) -> None:
                 raise ValueError(
                     f"{self.quantization} quantization is currently not "
                     f"supported in ROCm.")
-            if self.quantization != "marlin":
+            if (self.quantization != "marlin"
```

Comment:
Can we rewrite this as `self.quantization not in ["marlin", "gptq_marlin"]`? It is easier to read and also we will add more quantization formats here soon 😃